presto-hive/src/test/java/com/facebook/presto/hive/TestCteExecution.java

```diff
@@ -1261,9 +1257,24 @@ protected Session getSession()
     protected Session getMaterializedSession()
     {
         return Session.builder(super.getSession())
+                .setSystemProperty(VERBOSE_OPTIMIZER_INFO_ENABLED, "true")
                 .setSystemProperty(PUSHDOWN_SUBFIELDS_ENABLED, "true")
                 .setSystemProperty(CTE_MATERIALIZATION_STRATEGY, "ALL")
                 .setSystemProperty(CTE_FILTER_AND_PROJECTION_PUSHDOWN_ENABLED, "true")
                 .build();
     }
+
+    public static class CTEInfoValue
+    {
+        String name;
+        int frequency;
```

Comment:
Can you please add a materialized boolean here? Just in case we might want to assert non materialized ctes